Dockerfile

```diff
@@ -26,11 +26,6 @@ RUN apt-get update -qq \
  # Create the build user \
  && useradd --create-home --system build \
  \
- # Install box \
- && wget https://github.com/humbug/box/releases/download/3.8.4/box.phar \
```

Comment:
`humbug/box` will be installed by Composer into the `vendor-bin/php-box/vendor` directory.